pkg/controllers/mcs/service_export_controller.go

```diff
@@ -446,8 +478,32 @@ func cleanupWorkWithEndpointSliceDelete(c client.Client, endpointSliceKey keys.F
 		return err
 	}
 
+	return cleanEndpointSliceWork(c, work.DeepCopy())
+}
+
+// TBD: Currently, the EndpointSlice work can be handled by both service_export_controller.go and endpointslice_collect_controller.go. To indicate this, we've introduced the label endpointslice.karmada.io/managed-by. Therefore,
+// if managed by both controllers, simply remove each controller's respective labels.
+// If managed solely by its own controller, delete the work accordingly.
+// This logic should be deleted after the conflict is fixed.
+func cleanEndpointSliceWork(c client.Client, work *workv1alpha1.Work) error {
+	controllers := util.GetLabelValue(work.Labels, util.EndpointSliceWorkManagedByLabel)
+	controllerSet := sets.New[string]()
+	controllerSet.Insert(strings.Split(controllers, "-")...)
```

Comment:
nits, may be this `"-"` can be extracted as a variable, and give it a clear variable name.

Reply:
This logic will be replaced with another way, is it necessary to do this?